### DISCUSSION TITLE
Go install

### DISCUSSION THREAD

**Discussion by etozhefifan** [go-chi/chi#896] [Q&A] [answered]
I may be dumb, but how can I install chi with `go install` since `go get -u ...` is deprecated?

**Comment by fnoopv** [comment]
There is something wrong with your understanding, `go get` for library, `go install` for  installing executables, [offical](https://go.dev/doc/go-get-install-deprecation)

**Comment by VojtechVitek** [comment, marked as answer]
Run `go get -u ...` inside of your Go folder. If you don't have a `go.mod` file created, run `go mod init` first.

**Reply by etozhefifan** [reply]
Thanks.
@fnoopv I really had a wrong understanding of packages in go.  Your answer also helped me
